test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -87,6 +87,36 @@ public async Task GivenMoreThanTenPatients_WhenSearchedWithSortParam_ThenPatient
             }
         }
 
+        [Fact]
+        [HttpIntegrationFixtureArgumentSets(dataStores: DataStore.SqlServer)]
+        public async Task GivenPatientsWithMissingBirthDates_WhenSearchedWithSortParamAndMissingModifierFalse_ThenThosePatientsShouldNotReturn()
+        {
+            var tag = Guid.NewGuid().ToString();
+            var patients = await CreatePaginatedPatients(tag);
+
+            patients[0].BirthDate = null;
+            patients[1].BirthDate = null;
```

Comment:
Why is this changing anything? The Patients have already been created and there isn't a line that creates new versions of the Patients.

Reply:
That was old code. Not relevant.